CRM/Campaign/Form/Gotv.php

```diff
@@ -137,7 +137,7 @@ public function validateIds() {
 
     $surveys = CRM_Campaign_BAO_Survey::getSurveys();
     if (empty($surveys)) {
-      $errorMessages[] = ts("Oops. It looks like no surveys have been created. <a href='%1'>Click here to create a new survey.</a>", [1 => CRM_Utils_System::url('civicrm/survey/add', 'reset=1&action=add')]);
+      $errorMessages[] = ts("It looks like no surveys have been created yet. <a href='%1'>Click here to create a new survey.</a>", [1 => CRM_Utils_System::url('civicrm/survey/add', 'reset=1&action=add')]);
```

Comment:
Since the string will need retranslating anyway, this might be a good time to also "fix" the `<a href`. See https://docs.civicrm.org/dev/en/latest/translation/#avoid-variables-inside-strings where it talks about hyperlinks, so

`<a %1>` and then the parameter is `[1 => 'href = "' . CRM_Utils_System::url('civicrm/survey/add', 'reset=1&action=add') . '"']

Reply:
Thanks. Force pushed c1f9439 to fix this